client/packages/common/src/ui/layout/tables/utils/ColumnDefinitionSetBuilder.ts

```diff
@@ -48,7 +48,8 @@ export type ColumnKey =
   | 'theirReference'
   | 'returnReason'
   | 'availableNumberOfPacks'
-  | 'numberOfPacksToReturn';
+  | 'numberOfPacksToReturn'
+  | 'numberOfPacksReturned';
```

Comment:
I am wondering why this was added here ?
Btw i wasn't a big fan of defining common column definition like this. Would have quite liked common definitions that allow for click through navigation from the place of consumption (like a method or just object, vs key like this)

Reply:
Are you meaning you'd recommend defining the columns I've added here differently (it looks like there's a few ways to do it, I just went with what was used in inbound shipment...) or just that you don't like how the column definition set up works in this repo?